### ISSUE TITLE
Implement formal logging system

### ISSUE THREAD

**Issue by BSchilperoort** [EcoExtreML/zampy#13]
For now we return (debugging) info to the user with print statements.

Eventually we should probably change the print statements to a more formal logging system.

_Originally posted by @BSchilperoort in https://github.com/EcoExtreML/zampy/pull/11#discussion_r1265349919_

**Comment by geek-yang** [comment]
Good point! And this logging system should output all error and warning messages in one file (e.g. like the `.out` file on snellius).

**Comment by BSchilperoort** [comment]
And depending on the logging level and debug level, print to terminal.
